source/initial-dns-seedlist-discovery/tests/txt-record-with-overridden-ssl-option.yml

```diff
@@ -0,0 +1,11 @@
+uri: "mongodb+srv://test5.test.build.10gen.cc/?ssl=false"
+seeds:
+    - localhost.test.build.10gen.cc:27017
+hosts:
+    - localhost:27017
+    - localhost:27018
+    - localhost:27019
+options:
```

Comment:
Be consistent with the structure of options.  Previously it was always a single document, but here's in an array of documents

Reply:
Oops, fixed.